x-pack/plugins/security_solution/cypress/integration/detection_alerts/cti_enrichments.spec.ts

```diff
@@ -155,7 +155,7 @@ describe('CTI Enrichment', () => {
     });
   });
 
-  describe('with additional indicators', () => {
```

Comment:
Can you add a comment referencing this PR?

Reply:
That seems to be SOP for these skips.

Reply:
cool, thanks for letting me know!